src/gondi.ts

```diff
@@ -359,12 +364,13 @@ export class Gondi {
       return {
         ...renegotiationInput,
         offerHash: offerHash ?? zeroHash,
+        signature: zeroHash,
```

Comment:
Do you remember if contract will omit signatura checking when verifying the sender? I don't want this change to risk any failure